### PR TITLE
typo fix in README

### PR DESCRIPTION
**Pull request by celsodantas**
Tiny little typo fix in the README for the partial refresh section.

cc @rafaelfranca =)

### CODE REVIEW

**Comment by arthurnn** [comment]
thanks

File: README.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ Any node with an `id` attribute can be partially replaced. If the `id` contains
 // Will change #flash, #comments, #comments:123
 Turbolinks.visit(url, change: 'comments')
 
-// Will change #flash, #comment_123
+// Will change #flash, #comment:123
```
Comment (fxn) [inline README.md]: Does it also need a "s"?

Comment (Thibaut) [inline README.md]: good catch

Comment (rafaelfranca) [inline README.md]: Fixed at https://github.com/rails/turbolinks/commit/12de3ececa875095476c9fee09e8eaecc006ea65

**Comment by rafaelfranca** [comment]
Thanks

**Pull request merged**